### DISCUSSION TITLE
🎯 Status Update

### DISCUSSION THREAD

**Discussion by TGlide** [melt-ui/melt-ui#1238] [Announcements]
👋 Hello all! I figure it's been a while since I did any announcement, and I've gotten plenty of questions wondering what the status of the project is, since it's activity from my side has dropped a bit.

The short of it all is, we've gone through countless ideas for the future of Melt. Framework agnostic, runes, different implementations, syntaxes, etc. Meanwhile you've all been amazing and endured on, with issues, pull requests, activity on Discord, all of it!

I faced a bit of burnout during it, and so that's why things have dropped for a while. But I'm confident now in the new direction for Melt. 

## New direction

I initially wanted Melt to be agnostic, but that proved to be too much to my tired i-like-having-a-life self. And other projects, such as Zag.js, have it covered! They're amazing, go check them out.

Meanwhile, I keep falling in love with Runes over and over again. I've been hard at work building [Runed](https://runed.dev), and it's been amazing. 

So I'm creating the *best* (IMO) DX for headless components in Svelte, bridging the flexibility we currently have, with a ultra-friendly DX I've been chasing, going all in on Runes.

This will mean a big refactor however, doing a lot of things from scratch. It also means there won't be feature parity with the current Melt from the get go. I want to get things *really* right, and get tons of feedback, before hammering tons of components.

I'm not ready to show things just yet, but I'm getting ready for a big, testable, preview! Keep an eye out.

### Upgrade path and breaking changes

This change won't affect the current version of Melt. Instead, I'll be deploying it under a different npm package, still under the Melt organization. Meaning, you don't have to worry about updating `@melt-ui/svelte` and suddenly, everything changed from the ground up.

## Current Issues and PRs

Due to things being refactored majorly, I've taken less notice to new issues and PRs. I'll try to get back to them, mostly in terms of fixes to continue supporting the current version of Melt while things aren't ready yet, but my main focus is on the new iteration.

Once again, thanks everyone for the continued support! 🍨

**Comment by arxpoetica** [comment]
Maybe even a project rename to avoid confusion?

After all, it's the way of Svelte.

Ractive -> Svelte
Sapper -> SvelteKit
MeltUI -> ?

**Reply by TGlide** [reply]
That was the idea initially, but I *love* the name Melt UI. We'll see!

**Reply by kizivat** [reply]
Hear me out:

"Melte",

or potentially:

"Smelte" (could be too similar to Svelte though)

Just releasing it into the wild.

**Comment by saturnonearth** [comment]
I am probably biased (since I love Svelte) - but I think this is a better move. As cool as framework agnostic libraries are - I think that it just adds too many things to manage and keep track of. Keeping as close to the grain as possible with Svelte 5 is a great move. We all appreciate your hard work - no need to rush!

**Comment by herkulano** [comment]
This is amazing news! I love that you’re focusing on Svelte, there are plenty of libraries for other frameworks, especially React. Svelte really needs a high quality headless UI library. Thank you so much for pushing this forward 🙏

**Comment by Rykuno** [comment]
This is great news and entirely the right direction imho. Svelte 5 is a massive leap in the right direction of web frameworks; so having some breaking changes in meltUI to accompany those changes and improve DX is fine by me.

**Comment by itskazemk** [comment]
I just wanted to say great work you guys, specially your Calendar and date components are awesome (Localization rocks!). can't wait for your next-gen.
by the way I'm using you for Persian(Farsi) projects.
